src/components/SignupForm.js

```diff
@@ -57,6 +58,26 @@ const SignupForm = ({ className }) => {
             />
           </label>
         </div>
+        <div className="hidden">
+          <input
+            id="share"
+            type="checkbox"
+            aria-hidden="true"
+            className="mr-2.5 rounded h-5 w-5 relative top-1 sm:static focus:ring-secondary focus:border-secondary text-secondary mb-16"
+            onChange={() => {
+              setIsHiddenChecked(!isHiddenChecked)
+            }}
+          />
+
+          <label htmlFor="share">
+            <span
+              data-button-label
+              dangerouslySetInnerHTML={{
+                __html: sanitizeHtml("Do you agree to share your contact information with YYJ Tech."),
+              }}
+            />
```

Comment:
Just a question - why the `dangerouslySetInnerHtml` instead of just embedding a string? 
```suggestion
            <span data-button-label>Do you agree to share your contact information with YYJ Tech?</span>
```

Reply:
I actually don't know - the person that built this originally did that and I was just copying the pattern #lazy